JASP-Desktop/analysisforms/SummaryStatistics/summarystatscorrelationbayesianpairsform.ui

```diff
@@ -440,7 +440,7 @@
       <item row="0" column="0">
        <widget class="BoundGroupBox" name="correlationCoefficient">
         <property name="title">
-         <string>Correlation Coefficient</string>
+         <string>Sample Correlation Coefficient</string>
```

Comment:
Akash please also change the radio button "Pearson's rho" to "Pearson's r"

Reply:
Done